store/v2/root/factory.go

```diff
@@ -109,6 +109,8 @@ func CreateRootStore(opts *FactoryOptions) (store.RootStore, error) {
 			return nil, err
 		}
 		ssDb, err = rocksdb.New(dir)
+	default:
+		return nil, fmt.Errorf("unknown storage type: %s", opts.Options.SSType)
```

Comment:
Should this be split in another PR?

Reply:
minor cleanup, i think its fine but happy to move it if you think its beter

Reply:
works for me! didn't expect so see store changes there that's why.